### PR TITLE
Updated landing page to contain more information.

### PR DESCRIPTION
**Pull request by zivy**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
-(Linux, OS X, and Windows).
+SimpleITK is an image-analysis toolkit, providing a simplified programming interface to the algorithms and data structures of the Insight Segmentation and Registration Toolkit. It supports image operations on 2D, 3D and 4D multi-channel images. The toolkit is available in eight programming languages including Python, R, Java, C#, C++, Lua, Ruby, and TCL.
+
+SimpleITK includes several hundred classes providing a broad set of tools required for image analysis. The toolkit provides extensive support for image input and output, supporting more than 20 image file formats. A large number of filtering operations are also available, from basic image analysis operations such as edge detection, binary and grayscale morphological operations, and distance transformations to MRI bias field correction. Additionally, a variety of traditional operators used for image segmentation are available including region growing, connected component labelling, and the watershed transformation. Finally, the toolkit includes a robust registration framework, readily enabling one to align intra and inter modality images, supporting both rigid and non-rigid transformation models.
```
Comment (dave3d) [inline index.rst]: 'labelling' is the British spelling.  The rest of SimpleITK uses 'labeling', the American spelling.  I'd suggest going with the American spelling, mainly to be consist.

File: index.rst [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+
+SimpleITK is widely used by researchers from various domains requiring images analysis capabilities. Amongst others these include analysis of anatomical structures imaged with CT,MR, and PET, and analysis of cellular structures imaged using dual photon microscopy, focused ion beam scanning electron microscopy and focal plane array microscopy. Additionally, the toolkit has been used at a number of academic institutions as a tool for teaching medical image analysis.
```
Comment (dave3d) [inline index.rst]: image analysis (singular).

Beginning of the 2nd sentence is awkward and unclear.  Maybe something like 'Applications of SimpleITK have included the analysis of ...'

Or maybe 'These domains include ...'

IMO 'Amongst others' isn't needed, and 'these' needs to refer directly to something.

Comment (zivy) [reply index.rst]: Based on comments and discussion with @blowekamp, PR was completely revised.

**Comment by blowekamp** [comment]
The relevant information has been placed on the simpleitk.org site.

**Pull request closed without merging**